src/client.cpp

```diff
@@ -930,13 +930,15 @@ void Client::Send(NetworkPacket* pkt)
 }
 
 // Will fill up 12 + 12 + 4 + 4 + 4 bytes
-void writePlayerPos(LocalPlayer *myplayer, NetworkPacket *pkt)
+void writePlayerPos(LocalPlayer *myplayer, ClientMap *clientMap, NetworkPacket *pkt)
 {
 	v3f pf         = myplayer->getPosition() * 100;
 	v3f sf         = myplayer->getSpeed() * 100;
 	s32 pitch      = myplayer->getPitch() * 100;
 	s32 yaw        = myplayer->getYaw() * 100;
 	u32 keyPressed = myplayer->keyPressed;
+	s32 fov        = clientMap->getCameraFov() * 100;
+	s32 wanted_range    = clientMap->getControl().wanted_range;
```

Comment:
Will do.

Also, this would be the place to request the max distance allowable by the server when range_all is selected.